### PR TITLE
Fix: SettingsControls UI

### PR DESCRIPTION
**Pull request by 0x5bfa**
## Fixes

- [x] Fix an issue where SettingsExpander's inner Grid's CornerRadius was 0px.
  image
- [ ] Fix an issue where border will disapper on hovering clickable SettingsCard inside SettingsExpander
  image
- [ ] #219
  image

## PR Type

What kind of change does this PR introduce?

- Bugfix
- Code style update (formatting)

## PR Checklist

Please check if your PR fulfills the following requirements: 

- [x] Created a feature/dev branch in your fork (vs. submitting directly from a commit on main)
- [x] Based off latest main branch of toolkit
- [ ] Tested code with current supported SDKs
- [ ] New component
  - [ ] Documentation has been added
  - [ ] Sample in sample app has been added
  - [ ] Analyzers are passing for documentation and samples
  - [ ] Icon has been created (if new sample) following the [Thumbnail Style Guide and templates](https://github.com/CommunityToolkit/WindowsCommunityToolkit-design-assets)
- [ ] Tests for the changes have been added (if applicable)
- [ ] Header has been added to all new source files
- [x] Contains **NO** breaking changes

### CODE REVIEW

**Comment by 0x5bfa** [comment]
@dotnet-policy-service agree

**Comment by 0x5bfa** [comment]
@niels9001 Since XamlStyer always style XAML every time I saved files with Ctrl+S, the diff includes formatting changes. Are you ok with this?

**Comment by niels9001** [comment]
> @niels9001 Since XamlStyer always style XAML every time I saved files with Ctrl+S, the diff includes formatting changes. Is it ok?

Could you try and run `ApplyXamlStyling.ps1` and push the changes? That should clear that up :).

Like:

`cd D:\Windows
.\ApplyXamlStyling.ps1 -main`

File: components/SettingsControls/src/SettingsExpander/SettingsExpander.xaml [hunk 1]
```diff
@@ -93,7 +92,7 @@
                                                     IsClickEnabled="False" />
                             </muxc:Expander.Header>
                             <muxc:Expander.Content>
-                                <Grid>
+                                <Grid CornerRadius="{TemplateBinding CornerRadius}">
```
Comment (niels9001) [inline components/SettingsControls/src/SettingsExpander/SettingsExpander.xaml:95]: When setting the `CornerRadius` to `24`, this now happens:
image

Comment (niels9001) [reply components/SettingsControls/src/SettingsExpander/SettingsExpander.xaml:95]: @0x5bfa would it make sense to add a `Converter` here that translates the `CornerRadius`? E.g. a `CornerRadius` of `24` would be converted to `24,0,0,24`?

Comment (0x5bfa) [reply components/SettingsControls/src/SettingsExpander/SettingsExpander.xaml:95]: Yes definitely, I'd like to make it more generally so everyone can use everywhere. Thus, using ConverterParameter like 'ConverterParameter=1001' true and false combination - it means 24 0 0 24 according to the input.

**Comment by niels9001** [comment]
@0x5bfa FYI, fixed the wrapping issue here: https://github.com/CommunityToolkit/Windows/pull/252

**Comment by 0x5bfa** [comment]
I didn't think how troublesome styling from terminal. I'm sorry, I'm gonna close.

I hope that someone quickly fix.

**Pull request closed without merging**